ingestion/functions/parsing/riograndedosul/riograndedosul.py

```diff
@@ -0,0 +1,246 @@
+import os
+import sys
+from datetime import date, datetime
+import csv
+
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+try:
+    import parsing_lib
+except ImportError:
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'common/python'))
+    import parsing_lib
+
+_AGE_INDEX = 5
+_GENDER_INDEX = 4
+_ETHNICITY = 20
+_COUNTY = 1
+_DATE_CONFIRMED = 7
+_DATE_SYMPTOMS = 8
+_HOSPITALIZED = 11
+_DATE_DEATH = 19
+_FEVER = 12
+_COUGH = 13
+_SORETHROAT = 14
+_SHORTBREATH = 15
+_OTHER = 16
+_COMORBIDITIES = 17
+_INDIGENOUS = 21
+_NEIGHBORHOOD = 24
+
+_COMORBIDITIES_MAP = {
+    'Doenças renais crônicas em estágio avançado (graus 3, 4 ou 5)': 'chronic kidney disease',
+    'Asm': 'asthma',
+    'Doença Cardiovascular Crônica': 'cardiovascular disease',
+    'Doenças cardíacas crônicas': 'heart disease',
+    'Doença Hematológica Crônic': "hematopoietic system disease",
```

Comment:
Following a conversation with Moritz (see Paraiba PR - now closed), Doença Hematológica Crônica directly translates to hematologic disease, which is not in the DO database. Hematopoietic system disease is actually subtly different. As a result I would put this in notes, if it is present.